### PR TITLE
add coverage to `local` commands

### PR DESCRIPTION
**Pull request by bmc-msft**
Based off of #1090.   

For a simplified view, try this link: 

https://github.com/bmc-msft/onefuzz/compare/use-enums-for-local-command-names...add-coverage-subcommand

### CODE REVIEW

File: src/agent/onefuzz-agent/src/local/coverage.rs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    let target_timeout = value_t!(args, TARGET_TIMEOUT, u64).ok();
+    let coverage_filter = value_t!(args, TARGET_TIMEOUT, String).ok();
+
+    let readonly_inputs = if local_job {
+        vec![
+            get_synced_dir(INPUTS_DIR, common.job_id, common.task_id, args)?
+                .monitor_count(&event_sender)?,
+        ]
+    } else {
+        get_synced_dirs(READONLY_INPUTS, common.job_id, common.task_id, args)?
```
Comment (ranweiler) [inline src/agent/onefuzz-agent/src/local/coverage.rs:41]: Double-checking: we really don't want to also include `INPUTS_DIR` in this branch?

**Review by chkeita — APPROVED** [review]

**Review by ranweiler — APPROVED** [review]

File: src/agent/onefuzz-agent/src/local/coverage.rs [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        )
+    } else {
+        args.push(
+            Arg::with_name(READONLY_INPUTS)
+                .takes_value(true)
+                .required(true)
+                .long(READONLY_INPUTS)
+                .multiple(true),
+        )
+    }
```
Comment (bmc-msft) [inline src/agent/onefuzz-agent/src/local/coverage.rs:126]: (highlighting for other comment)

File: src/agent/onefuzz-agent/src/local/coverage.rs (hunk 1 above)
Comment (bmc-msft) [reply src/agent/onefuzz-agent/src/local/coverage.rs:41]: This mirrors the libfuzzer-coverage directly.

Note, the arg builder makes it such that INPUTS and READONLY_INPUTS are used differently depending on the caller.

Comment (bmc-msft) [reply src/agent/onefuzz-agent/src/local/coverage.rs:41]: See the block of code from this PR I commented on with "(highlighting for other comment)".

**Pull request merged**